core/src/main/java/tech/tablesaw/api/Table.java

```diff
@@ -864,35 +864,87 @@ public Table dropWhere(Selection selection) {
     return newTable;
   }
 
+
   /**
+   * Returns a new column, where the first n columns are the groupingColumns. There are then p additional
+   * columns, which is the product of each unique value in the pivot column and aggregatedColumn. The 
+   * values in each of the cells in these new columns are the result of applying the given AggregateFunction 
+   * to the data in each of aggregatedColumn, grouped by the values of groupingColumn and pivotColumn.
+   * 
+   * If more than one aggregatedColumn is provided then each is appended to each unique value of the pivot
+   * column in the format "{PivotColumnValue}.{AggregatedColumnName}
+   * 
+   * @param groupingColumn
+   * @param pivotColumn
+   * @param aggregatedColumn
+   * @param aggregateFunction
+   * @return
+   */
+  public Table pivot(
+    List<CategoricalColumn<?>> groupingColumns,
+    CategoricalColumn<?> pivotColumn,
+    List<NumericColumn<?>> aggregatedColumns,
+    AggregateFunction<?, ?> aggregateFunction) {
+    return PivotTable.pivot(this, groupingColumns, pivotColumn, aggregatedColumns, aggregateFunction);
+  }
+
+  /**
+   * Returns a new column, where the first n columns are the groupingColumns. There are then p additional
+   * columns, which is the product of each unique value in the pivot column and aggregatedColumn. The 
+   * values in each of the cells in these new columns are the result of applying the given AggregateFunction 
+   * to the data in each of aggregatedColumn, grouped by the values of groupingColumn and pivotColumn.
+   * 
+   * If more than one aggregatedColumn is provided then each is appended to each unique value of the pivot
+   * column in the format "{PivotColumnValue}.{AggregatedColumnName}
+   * 
+   * @param groupingColumnNames
+   * @param pivotColumnName
+   * @param aggregatedColumnNames
+   * @param aggregateFunction
+   * @return
+   */
+  public Table pivot(
+      List<String> groupingColumnNames,
+      String pivotColumnName,
+      List<String> aggregatedColumnNames,
+      AggregateFunction<?, ?> aggregateFunction) {
+        return pivot(
+            groupingColumnNames.stream().map(this::categoricalColumn).collect(Collectors.toList()),
```

Comment:
Codacy is complaining that you're doing `Collectors.toList` here when `toList` is already statically imported. I'm fine either way to want it as long as it's consistent